app.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "workshop-portal",
   "scripts": {
+    "postdeploy": "bundle exec rake db:schema:load db:seed db:populate_sample_data"
```

Comment:
This is called *once* on a *review app* whenever a new merge request - and a review app - is *created*.